v20.2/drop-user.md

```diff
@@ -10,6 +10,10 @@ The `DROP USER` [statement](sql-statements.html) removes one or more SQL users.
  Since the keywords `ROLE` and `USER` can now be used interchangeably in SQL statements for enhanced Postgres compatibility, `DROP USER` is now an alias for [`DROP ROLE`](drop-role.html).
 {{site.data.alerts.end}}
 
+## Consideration
+
+Users that [own objects](authorization.html#object-ownership) cannot be dropped until the [ownership is transferred to another user](owner-to.html#change-a-databases-owner).
```

Comment:
Suggest "own objects such as databases or tables" or similar, for reasons mentioned elsewhere.